packages/api/src/car.js

```diff
@@ -209,18 +224,16 @@ async function addToCluster (car, env) {
  */
 
 /**
- * Backup given Car file keyed by /raw/${rootCid}/${userId}/${carHash}.car
+ * Upload given CAR file keyed by /raw/${rootCid}/${userId}/${carHash}.car
+ *
+ * @param {import('@aws-sdk/client-s3').S3Client} s3
+ * @param {string} bucketName
  * @param {Blob} blob
  * @param {CID} rootCid
  * @param {string} userId
- * @param {import('./env').Env} env
  * @param {('Unknown' | 'Partial' | 'Complete')} structure The known structural completeness of a given DAG.
  */
-async function backup (blob, rootCid, userId, env, structure = 'Unknown') {
-  if (!env.s3Client) {
-    return undefined
-  }
-
+async function uploadToBucket (s3, bucketName, blob, rootCid, userId, structure = 'Unknown') {
```

Comment:
minor: might be clearer here to change the function signature to take the string form of the `sourceCid`, as we're uploading a user generated CAR and we want to key it on the string form of the rootCid in the format provided in the CAR file.
```suggestion
async function uploadToBucket (s3, bucketName, blob, sourceCid, userId, structure = 'Unknown') {
```